contents/blog/the-posthog-array-1-32-0.md

```diff
@@ -0,0 +1,133 @@
+excerpt: Excerpt coming soon.
+---
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Do not upgrade to this version if you have deployed PostHog using Postgres. PostHog no longer supports Postgres as of <code>v1.31.0</code> (last version supported is <code>1.30.0</code>) and you must <a href="/docs/self-host/migrate-from-postgres-to-clickhouse" target="_blank">upgrade to ClickHouse</a> first.
+</blockquote>
+
+## PostHog 1.32.0 release notes
+
+> Don't see the new features on your self-hosted deployment? Remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Redesigned Persons & Groups pages](#redesigned-persons--groups-pages)
+- [Short link sharing](#short-link-sharing)
+- [Improved insight experience](#improved-insight-experience)
+- [Funnels vertical breakdown](#funnels-vertical-breakdown)
+
+### Redesigned Persons & Groups pages
+
+We've redesigned how the Persons & Groups pages look to make it easier to find what you're looking for. You'll now be able to easily see which groups (if any) the user belongs to, and better manage the user's properties.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_32_0-new-person-page.gif" alt="Example screenshot: New Person page" />
+
+<br />
+
+### Short link sharing
+
+When sharing saved insights, you will now get a short and sweet URL to share with your teammates. This means something like this: `https://app.posthog.com/insights/y3tskB1B`, instead of well.... a very, very long URL you would see before. Further, this now makes it possible to create very complex insights with multiple series and filters.
+
+### Improved insight experience
+
+Searching for the right event or property when building an insight could be hard. That's why we implemented some key improvements to make it easier:
+1. You will now see a "Stale" label next to any event we haven't seen for a while, so you know not to use it.
+2. When filtering events by properties, we will now clearly show you properties that are relevant for the event (as opposed to all your properties).
+3. When filtering events by properties, we will now give you better suggestions about potential values to filter on.
+
+In addition to the above, we've improved how tooltips are displayed so they contain clearer and more useful information, and implemented a frequently asked for "Legend" feature. You can now share screenshots with all the required information to understand them.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_32_0-new-tooltips.png" alt="Example screenshot: New tooltips" />
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_32_0-legends.png" alt="Example screenshot: Legends" />
+
+### Funnels vertical breakdown
+
+Funnels with breakdowns just got a lot better. This new visualization can help you better understand how your users are converting.
```

Comment:
Also kind of fluffy, what specifically makes it better so that I want to upgrade? I don't feel like the screenshot supports this, and in any case there's no before/after so it's hard to make the comparison myself.